src/WebApiContrib.Core.Formatter.MessagePack/MessagePackFormatterOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MessagePack;
+using MessagePack.Resolvers;
+using System.Collections.Generic;
+
+namespace WebApiContrib.Core.Formatter.MessagePack
+{
+    public class MessagePackFormatterOptions
+    {
+        public IFormatterResolver FormatterResolver { get; set; } = ContractlessStandardResolver.Instance;
+
+        public HashSet<string> SupportedContentTypes { get; set; } = new HashSet<string> { "application/x-msgpack" };
```

Comment:
I think using `x-*` media types is generally frowned upon these days. We've been using `application/msgpack` for a while...